src/__integrationtests__/Attestation.spec.ts

```diff
@@ -0,0 +1,256 @@
+/**
+ * @group integration
```

Comment:
Is this annotation jest specific?

Reply:
I'm using [jest-runner-groups](https://www.npmjs.com/package/jest-runner-groups) to be able to run the integration tests separately